### PR TITLE
Fix some NaN, sharding, presence errors and everything

### PR DESCRIPTION
**Pull request by Doges**
Hii~~

I fixie wixied the JSONapi fucksy wucksie!!! xpp

I repwaced the Prototypes.func wiff a more ~streamylined~ kawaii Pwototypiee.funkywunks object that make the JS go sooper dooper fast!! xD

starts twerking

Can u pwease merge my pwull wequest senpai?!! UwU

Cwoses #1283

### CODE REVIEW

File: src/http/api/contributors.js [hunk 1]
```diff
@@ -10,10 +10,12 @@ module.exports = class Contributors extends Route {
...
   register (app) {
     const router = Router()
-
+    // what the fuck is happening here
+    // TODO: everything, it sucks
     router.get('/', async (req, res) => {
       const guild = this.client.guilds.cache.get(process.env.BOT_GUILD)
-      const roles = guild.roles
+      const roles = guild.roles.cache
+      console.log(roles)
```
Comment (moruzerinho6) [inline src/http/api/contributors.js:18]: Someone forgot to remove console log?

Comment (almeidx) [reply src/http/api/contributors.js:18]: that part of the code still needs to be refactored to support sharding

Comment (Doges) [reply src/http/api/contributors.js:18]: check line 13

**Comment by Doges** [comment]
fixed the PR description

**Comment by Doges** [comment]
twitch was also glitched, fixed on commit #61b9eb3

File: src/apis/GoogleSearch.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 const { APIWrapper } = require('../')
 const fetch = require('node-fetch')
 
-module.exports = class CrowdinAPI extends APIWrapper {
+module.exports = class GoogleSearchAPI extends APIWrapper {
```
Comment (almeidx) [inline src/apis/GoogleSearch.js:4]: image

**Comment by jaxxibae** [comment]
May I ask what does this fix? I didn't understand that much what does this PR do

**Comment by Doges** [comment]
DiscordCanary_vcVIgIcXNv


> 
> 
> May I ask what does this fix? I didn't understand that much what does this PR do

**Comment by Doges** [comment]
that won't fix anything now. closing it

**Pull request closed without merging**